core/src/main/scala/org/broadinstitute/dsp/workbench/hamm/dao/CostDbDao.scala

```diff
@@ -1,14 +1,13 @@
-package org.broadinstitute.dsp.workbench.hamm.service
+package org.broadinstitute.dsp.workbench.hamm
 
 import org.broadinstitute.dsp.workbench.hamm.db._
 import org.broadinstitute.dsp.workbench.hamm.model._
 import org.broadinstitute.dsp.workbench.hamm.model.HammException
-import org.broadinstitute.dsp.workbench.hamm.HammLogger
 import org.broadinstitute.dsp.workbench.hamm.auth.SamAuthProvider
 import org.http4s.Credentials.Token
 import org.http4s.Status
 
-class CostService(samAuthProvider: SamAuthProvider, dbRef: DbReference, jobTable: JobTableQueries, workflowTable: WorkflowTableQueries) extends HammLogger {
+class CostDbDao(samAuthProvider: SamAuthProvider, dbRef: DbReference, jobTable: JobTableQueries, workflowTable: WorkflowTableQueries) extends HammLogger {
```

Comment:
renaming this because in http4s doc, xxservice is usually what provides the `HttpRoutes`.

Reply:
But is this really the "DbDao" - aren't JobTable and WorkflowTable classes the data access objects for the db?